packages/drivers/redis/src/actor.ts

```diff
@@ -10,6 +10,12 @@ export class RedisActorDriver implements ActorDriver {
         this.#redis = redis;
     }
 
+    get platform(): ActorPlatform {
```

Comment:
I think your implementation is correct, but we might consider changing the name:

`ActorDriver` != the platform. For example, the Redis is going to be used for both Node.js, Bun, and the upcoming Vercel platform (source is out of date, but can be found [here](https://github.com/rivet-gg/actor-core/pull/637/files#diff-f824297c2ee7b978bbc80d3d9f53943e32c4b8f1ecb77d09a37ebebcde7e0caf)).

I'm wondering if we should name it something other than `platform` since "platform" implies Rivet/Cloudflare/Node/etc. If I were describing this, I'd call it "actor driver context" so maybe something like `c.driver`? I'm concerned "driver" may be too far removed from how the developer is thinking about it.

We can discuss this in more depth in our driver docs: https://actorcore.org/drivers/overview

Reply:
This ultimately lead me to rethink the whole approach. I took @NathanFlurry's [idea](https://discord.com/channels/822914074136018994/1353803015265718366/1353912492677398528) and implemented a simpler version. 

Now drivers have a `context` getter. It returns `unknown`, leaving that up to the driver to decide. This is passed to `createVars` but is otherwise left out of the context. 

I've removed the interface for now. There's no mechanism for switching on the driver name as previously was implemented. I think it would be better to think that interface through more. I can certainly re-add it (or something similar) if y'all think it's necessary though.